src/sentry/incidents/grouptype.py

```diff
@@ -107,3 +107,8 @@ class MetricAlertFire(GroupType):
     @classmethod
     def allow_post_process_group(cls, organization: Organization) -> bool:
         return features.has("organizations:workflow-engine-metric-alert-processing", organization)
+
+
+# This needs to be removed once the import in getsentry is updated to use MetricIssue
+class MetricAlertFire:
+    pass
```

Comment:
this can be removed once [this pr](https://github.com/getsentry/getsentry/pull/17255) lands